draft-ietf-ntp-roughtime.md

```diff
@@ -258,8 +262,27 @@ The value of the NONC tag is a 32 byte nonce. It SHOULD be generated
 in a manner indistinguishable from random. BCP 106 contains specific
 guidelines regarding this {{!RFC4086}}.
 
+### SRV
+
+The SRV tag is used by the client to indicate which long-term public
+key it expects to verify the response with. The value of the SRV tag
+is `H(0xff || dns_name)` where `dns_name` is the serer's DNS name.
```

Comment:
I'm not sure we want a 1:1 mapping of key to DNS name. Wasn't the idea here to have multiple keys on the same DNS name? Also typo in server

Reply:
Fixed the typo.

From CF's perspective, the goal is be able to support multiple long-term keys on the same IP and port. It's fine if the mapping from long-term public keys to DNS names is 1:1.

Also, matches your suggestion here: https://mailarchive.ietf.org/arch/msg/ntp/K1InWKz-VKdL20KgSqj6AV0p4cs/

Reply:
I listed that more as a possibility than something to do

Reply:
I think 1:1 mapping is perfectly fine, and I don't mind using the DNS name this way. If you don't think this would work, I think the best alternative is to replace `dns_name` input to the hash with the long-lived public key.

What would you like to do? Do you have a suggestion for an alternative?

Reply:
How about hash the public key? I prefer that, but we might have to go to the list for consensus

Reply:
Either works for me. We previously discussed hashing the public key as being potentially problematic from a fingerprinting standpoint, but tbh I don't think we're all that much better off with the DNS name. (A malicious roughtime server can always advertise DNS names to specific clients.)

Reply:
Tacked on a commit for this, PTAL.